addons/mail_group/controllers/portal.py

```diff
@@ -184,8 +194,8 @@ def group_message_get_replies(self, group, message, last_displayed_id, **post):
     # SUBSCRIPTION
     # ------------------------------------------------------------
 
-    @http.route('/groups/subscription', type='json', auth='public', website=True)
-    def group_subscribe(self, group_id=0, action='subscribe', email=None, token=None, **kw):
+    @http.route('/groups/subscribe', type='json', auth='public', website=True)
```

Comment:
For `/groups/subscribe` is not possible because you want to split the route subscribe / unsubscribe and also old token won't be valid (because it used "website_mail_channel" as key)

Ok to remove the "S" and to split the "subscribe-confirm" route